spring-cloud-azure-samples/spring-cloud-azure-config-sample/src/main/resources/bootstrap.properties

```diff
@@ -1,4 +1,3 @@
 spring.cloud.azure.config.connection-string=put-your-connection-string-here
```

Comment:
I just want to confirm whether we really need to rely on connections string. As all other spring cloud modules don't have such low level dependency, they only rely on logical concepts. And our design goal is to minimize such low level details.

Reply:
Will improve if possible in the future.